pkgs/development/python-modules/tinyarray/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  numpy,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "tinyarray";
+  version = "1.2.4";
+  format = "pyproject";
```

Comment:
```suggestion
  pyproject = false;
```

Reply:
I guess you want to change it to `pyproject = true;`.

Reply:
I guess you want to change it to `pyproject = true;`.